src/App.tsx

```diff
@@ -14,11 +14,160 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  showList: boolean,
+  isReversed: boolean,
+  sortBy: string,
+  minLength: number,
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    showList: false,
+    isReversed: false,
+    sortBy: '',
+    minLength: 0,
+  };
+
+  showData = () => {
```

Comment:
this line ends up in error: 
`Parse errors in imported module './App': Expression expected. (32:2)`
please help to handle it to enable deploying to the [DEMO LINK](https://meforoma.github.io/react_list-of-goods/)